lib/facility_factory.rb

```diff
@@ -0,0 +1,69 @@
+class FacilityFactory
+  attr_reader :operating_facilities
+
+  def initialize
+    @operating_facilities = []
```

Comment:
I like that you created an instance variable here to store the data in! That way we only have to run the method below once.